api/neuronaApp/views/space_view.py

```diff
@@ -0,0 +1,88 @@
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework import status
+from ..models import Spaces
+from ..serializers import SpaceSerializer
+
+# Get all spaces
+
+
+@api_view(['GET'])
+def get_spaces(request):
+    try:
+        spaces = Spaces.objects.all()
+        serializer = SpaceSerializer(spaces, many=True)
+        return Response(serializer.data)
+    except Exception as e:
+        return Response({'error': 'Server error: ' + str(e)}, status=status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
There should not be any raised exception and the error detail must not be, under any circumstances, be returned, especially in a production environment !  
```suggestion
    spaces = Spaces.objects.all()
    serializer = SpaceSerializer(spaces, many=True)
    return Response(serializer.data)
```